userspace/libscap/linux/scap_ppm_sc.c

```diff
@@ -0,0 +1,551 @@
+	[PPME_SYSCALL_OPEN_E] = (ppm_sc_code[]){PPM_SC_OPEN, -1},
+	[PPME_SYSCALL_OPEN_X] = (ppm_sc_code[]){PPM_SC_OPEN, -1},
+	[PPME_SYSCALL_CLOSE_E] = (ppm_sc_code[]){PPM_SC_CLOSE, -1},
+	[PPME_SYSCALL_CLOSE_X] = (ppm_sc_code[]){PPM_SC_CLOSE, -1},
+	[PPME_SYSCALL_READ_E] = (ppm_sc_code[]){PPM_SC_READ, -1},
+	[PPME_SYSCALL_READ_X] = (ppm_sc_code[]){PPM_SC_READ, -1},
+	[PPME_SYSCALL_WRITE_E] = (ppm_sc_code[]){PPM_SC_WRITE, -1},
+	[PPME_SYSCALL_WRITE_X] = (ppm_sc_code[]){PPM_SC_WRITE, -1},
+	[PPME_SYSCALL_BRK_1_E] = (ppm_sc_code[]){PPM_SC_BRK, -1},
+	[PPME_SYSCALL_BRK_1_X] = (ppm_sc_code[]){PPM_SC_BRK, -1},
+	[PPME_SYSCALL_EXECVE_8_E] = (ppm_sc_code[]){PPM_SC_EXECVE, -1},
+	[PPME_SYSCALL_EXECVE_8_X] = (ppm_sc_code[]){PPM_SC_EXECVE, -1},
+	[PPME_SYSCALL_CLONE_11_E] = (ppm_sc_code[]){PPM_SC_CLONE, -1},
+	[PPME_SYSCALL_CLONE_11_X] = (ppm_sc_code[]){PPM_SC_CLONE, -1},
+	[PPME_PROCEXIT_E] = (ppm_sc_code[]){PPM_SC_SCHED_PROCESS_EXIT, -1},
+	[PPME_PROCEXIT_X] = NULL,
```

Comment:
Philosophical question (I know we can't change this): wouldn't PROCEXIT_X be more appropriate? :D

Reply:
> PROCEXIT_X

I don't get it ahha isn't it already 'PPME_PROCEXIT_X`? :laughing: 
Or you mean to rename PPM_SC_SCHED_PROCESS_EXIT?